paymentsheet/src/main/java/com/stripe/android/link/theme/Color.kt

```diff
@@ -51,22 +52,27 @@ internal data class LinkColors(
     val textWhite: Color,
     val textBrand: Color,
     val textCritical: Color,
-    val textOnButtonPrimary: Color,
+    val onButtonPrimary: Color,
+    val onButtonBrand: Color,
     val iconPrimary: Color,
     val iconSecondary: Color,
     val iconTertiary: Color,
     val iconWhite: Color,
     val iconBrand: Color,
     val iconCritical: Color,
     val outline: Color,
-)
+) {
+    val buttonLinkBrand: Color get() = Brand200
+    val onButtonLinkBrand: Color get() = Neutral900
+}
 
 internal object LinkThemeConfig {
     fun colors(isDark: Boolean): LinkColors {
         return if (isDark) colorsDark else colorsLight
     }
 
     private val colorsLight = LinkColors(
+        isDark = false,
```

Comment:
👀 added to replace `isSystemInDarkTheme()` usages so `ALWAYS_DARK/LIGHT` is applied